code/game/objects/items/crayons.dm

```diff
@@ -147,6 +147,29 @@
 	colour = "#DA00FF"
 	colourName = "purple"
 
+/obj/item/toy/crayon/random/New()
+	icon_state = pick(list("crayonred", "crayonorange", "crayonyellow", "crayongreen", "crayonblue", "crayonpurple"))
+	switch(icon_state)

```

Comment:
I was wondering if white was only mime specific. Does it serve any amazing purpose besides being racially supreme and special only to the mime? I can't think of any other than writing "invisible" text if even. Can you even dye jumpsuits with it?